sites/all/modules/reol_lazy_image/reol_lazy_image.info

```diff
@@ -0,0 +1,5 @@
+name = eReolen lazy image
```

Comment:
Maybe we should call the module “eReolen lazy load images” (`reol_lazy_load_images`)?

Reply:
It should be Reol lazy images... it's for both sites. Fixed.

Reply:
Actually my suggestion was to call the module `reol_lazy_load_images`, i.e. with `lazy_load_images` in the name, since it adds lazy load(ing) of images.